### ISSUE TITLE
Tests for twisted.mail should conform to the new style of naming test classes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7708]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)| @eeshangarg reported|
|-|-|
|Trac ID|trac#7708|
|Type|enhancement|
|Created|2014-11-08 16:47:27Z|
|Branch|https://github.com/twisted/twisted/tree/mail-tests-names-7708|

This ticket is a follow-up to ticket [ticket:7032].

Attachments:

* [bugfix_7708.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/94b/94b894389c79c92f5b6c7304bd1353a7e69dc4f3/919891cbe2c03483ddd0bdf52a31f6e70a00663e.patch) (22605 bytes) - added by eeshangarg on 2014-11-08 17:13:55Z - 
* [bugfix_7708_2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/94b/94b894389c79c92f5b6c7304bd1353a7e69dc4f3/4966ba23ef64dd605541e4f5d5968b67602357e8.patch) (22712 bytes) - added by eeshangarg on 2014-11-17 12:30:20Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7708 7708
type__enhancement enhancement
reporter__eeshangarg eeshangarg
priority__normal normal
milestone__ 
branch__branches_mail_tests_names_7708 branches/mail-tests-names-7708
branch_author__eeshangarg eeshangarg
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__easy easy
time__1415465247699511 1415465247699511
changetime__1417037662971290 1417037662971290
version__None None
owner__exarkun exarkun
cc__eeshangarg
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_1"></a>|@eeshangarg set owner to @glyph|
|-|-|

Hi! I just added a patch that changes all of the tests in twisted.mail to use the new style of naming test classes as discussed in ticket [ticket:7032].

Thanks!
Eeshan

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph removed owner|
|-|-|

Un-assigning all review tickets assigned to me.  Unless there's a specific reason _I_ need to review something, let's leave them unassigned :-).

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

It looks good...just a missing news file

**Comment by twisted-trac** [comment]
|[<img alt="eeshangarg's avatar" src="https://avatars.githubusercontent.com/u/7251823?s=50" width="50" height="50">](https://github.com/eeshangarg)<a name="note_4"></a>|@eeshangarg commented|
|-|-|

Hello!

I just added a new patch called bugfix_7708_2.patch that includes the .misc news file. Thanks a lot for the review. :)

Regards,
Eeshan Garg

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

(In [43516]) Branching to 'mail-tests-names-7708'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

(In [43517]) Apply bugfix_7708_2.patch 

Refs [#7708](https://github.com/twisted/twisted/issues/7708)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

Thanks.  These changes look fine to me.  Buildbot is pretty happy too.  I'll merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set status to `closed`|
|-|-|

(In [43518]) Merge mail-tests-names-7708

Author: eeshangarg
Reviewer: exarkun
Fixes: [#7708](https://github.com/twisted/twisted/issues/7708)

Rename all of the test case classes in Twisted Mail to conform to the
recently documented test case class naming convention.
